mcoding_bot/cogs/misc.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import dotenv
+import os
+import pincer
+
+dotenv.load_dotenv()
```

Comment:
iirc, dotenv only needs to be loaded once in the program

Reply:
Yes but because of the structure of this bot there's no way to know which file is loaded first, so I have to load the dotenv everywhere. I might fix this later but that's a whole refactoring thing, not my fault.